rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -51,6 +52,24 @@ struct SubscriptionOptionsBase
   /// Optional RMW implementation specific payload to be used during creation of the subscription.
   std::shared_ptr<rclcpp::detail::RMWImplementationSpecificSubscriptionPayload>
   rmw_implementation_payload = nullptr;
+
+  // Options to configure topic statistics collector in the subscription
+  struct TopicStatisticsOptions
+  {
+    // Represent the state of topic statistics collector
+    enum class TopicStatisticsState {ENABLED, DISABLED};
+
+    // Enable and disable topic statistics calculation and publication. Defaults to disabled.
+    TopicStatisticsState state = TopicStatisticsState::DISABLED;
+
+    // Topic to which topic statistics get published when enabled. Defaults to /system_metrics.
+    std::string publish_topic = "system_metrics";
```

Comment:
This is inconsistent with the comment, is it supposed to be `/system_metrics` (absolute) or `system_metrics` (relative, subject to namespacing)?

Reply:
```suggestion
    std::string publish_topic = "/statistics";
```

Per http://wiki.ros.org/Topics#Topic_statistics